tests/model_serving/model_server/llmd/llmd_configs/config_models.py

```
@@ -47,6 +56,22 @@ class TinyLlamaS3GpuConfig(GpuConfig):
     storage_uri = ModelStorage.S3.TINYLLAMA
     model_name = ModelName.TINYLLAMA
 
+    @classmethod
+    def container_env(cls):
+        return super().container_env() + [
+            {
+                "name": "VLLM_ADDITIONAL_ARGS",
+                "value": "--enable-auto-tool-choice --tool-call-parser hermes",
+            },
+        ]
```

Ruff (0.16.0)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)